runtime/appruntime/api/handler_test.go

```diff
@@ -139,4 +144,40 @@ func TestDesc_EndToEnd(t *testing.T) {
 			}
 		})
 	}
+
+	testMetricsExporter.AssertCounter(t, "e_requests_total", 2, map[string]string{
+		"service":  "service",
+		"endpoint": "endpoint",
+	})
```

Comment:
The total number of requests is 2 because the auth handler (which would increase this counter) isn't called in this test. I think this is expected.